aws-operator-role/iam-policy.json

```diff
@@ -46,14 +43,12 @@
                 "kms:*",
                 "logs:*",
                 "route53:*",
-                "route53domains:*",
                 "s3:*",
                 "sts:AssumeRole",
                 "sts:DecodeAuthorizationMessage",
                 "sts:GetFederationToken",
                 "servicequotas:*",
-                "support:*",
-                "trustedadvisor:*"
```

Comment:
IIRC `trustedadvisor` is enabled only on some accounts. I think e.g. adidas is using this. I can be wrong here. @JosephSalisbury do you remember?

Reply:
Possibly you tested on account where we don't have it.

Reply:
I think we deactivated it on all installations recently. However, we might want to use it again in the future?

Reply:
If we really don't use it (including management clusters) then LGTM.

Reply:
it requires a specific level of support plan that only some accounts have. however, it's v useful when they do have it, because we can alert on whether we're getting close to service limits

i remember we had an issue w/ trusted advisor / talking with the TA team, but i don't remember the outcome - if we can still use it, we should

Reply:
Ok I will leave it in there for now, thanks 👍🏻 .